crates/matrix-sdk-base/src/store/traits.rs

```diff
@@ -50,9 +49,8 @@ use crate::{
 
 /// An abstract state store trait that can be used to implement different stores
 /// for the SDK.
-#[cfg_attr(target_arch = "wasm32", async_trait(?Send))]
-#[cfg_attr(not(target_arch = "wasm32"), async_trait)]
-pub trait StateStore: AsyncTraitDeps {
+#[async_trait]
+pub trait StateStore: fmt::Debug + Send + Sync {
```

Comment:
Pretty sure that this change breaks `matrix-sdk-indexeddb` (you can see that one of the CI jobs for it failed).

Reply:
You are absolutely right, I hadn't been considering that.

Reply:
I've restored this trait temporarily so the build keeps working.